warehouse/templates/base.html

```diff
@@ -51,7 +51,7 @@
 
     <link rel="stylesheet" href="{{ request.static_path('warehouse:static/dist/css/warehouse.css') }}">
     <link rel="stylesheet" href="{{ request.static_path('warehouse:static/dist/css/font-awesome.css') }}">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400italic,600,600italic,700,700italic|Source+Code+Pro:500">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400,600,600,700,700|Source+Code+Pro:500">
```

Comment:
I think we're repeating the font weights here, this line can probably just become:

```diff
- <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400,600,600,700,700|Source+Code+Pro:500">
+ <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700|Source+Code+Pro:500">